templates/db2server.j2.rsp

```diff
@@ -25,6 +25,7 @@ INSTALL_TYPE              = {{resp.install_type}}
 
 {%for inst in db2_instances %}
 
+{%if create_instances == true %}
```

Comment:
Why not just `{%if create_instances %}`?

Reply:
I prefer a clearer definition of conditionals where possible. The above suggestion would for example not work with following - `ansible-playbook -i hosts play.yml --diff --extra-vars "create_instances=false"` which is a bit odd as following will work as expected - `ansible-playbook -i hosts play.yml --diff --extra-vars '{"create_instances":false}'`. Also it will evaluate to `true` if it contains any random text except of 'no' and 'false' and maybe other keywords for false. So as it will be by default `true` it will be easier to change with anything to be considered as `false`. Let me know if that makes sense :)

Reply:
wow that's really weird. It's ok than, let's keep this way